tests/test_logging.py

```diff
@@ -134,38 +129,42 @@ def test_configure_extended_logging(self):
         }
         opsdroid.configure_logging(config)
         rootlogger = logging.getLogger()
-        self.assertEqual(len(rootlogger.handlers), 1)
-        self.assertEqual(logging.StreamHandler, type(rootlogger.handlers[0]))
+        assert len(rootlogger.handlers) == 1
+        assert isinstance(rootlogger.handlers[0], logging.StreamHandler)
 
     def test_configure_default_logging(self):
         config = {}
         opsdroid.configure_logging(config)
         rootlogger = logging.getLogger()
-        self.assertEqual(len(rootlogger.handlers), 2)
-        self.assertEqual(logging.StreamHandler, type(rootlogger.handlers[0]))
-        self.assertEqual(rootlogger.handlers[0].level, logging.INFO)
-        self.assertEqual(
-            logging.handlers.RotatingFileHandler, type(rootlogger.handlers[1])
-        )
-        self.assertEqual(rootlogger.handlers[1].level, logging.INFO)
-        self.assertLogs("_LOGGER", "info")
-
-
-class TestWhiteAndBlackFilter(unittest.TestCase):
-    def setUp(self):
-        self.config = {
-            "logging": {
-                "path": False,
-                "level": "info",
-                "console": True,
-                "filter": {"whitelist": ["opsdroid"], "blacklist": ["opsdroid.core"]},
-            }
+        assert len(rootlogger.handlers), 2
+        # TODO use isinstance
+        assert isinstance(rootlogger.handlers[0], logging.StreamHandler)
+        assert rootlogger.handlers[0].level == logging.INFO
+        assert isinstance(rootlogger.handlers[1], logging.handlers.RotatingFileHandler)
+
+        assert rootlogger.handlers[1].level == logging.INFO
+        # self.assertLogs("_LOGGER", "info")
```

Comment:
This one was checking if `_LOGGER` is using the info level, but now that I look at it, the previous assertion is doing exactly that, so we could just remove it

Reply:
I removed them